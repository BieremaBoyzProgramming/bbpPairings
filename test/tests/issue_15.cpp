// https://github.com/BieremaBoyzProgramming/bbpPairings/issues/15
void TEST_FUNCTION(const testing::Context &context)
{
  auto output_filename = STRINGIFY(TEST_ID) ".output";
  testing::run(
    context.exe_path.string()
    + " --burstein "
    + (context.data_folder_path / STRINGIFY(TEST_ID) ".input").string()
    + " -c > "
    + (context.data_folder_path / output_filename).string());
  testing::assert_file_content_matches(
    context.data_folder_path / output_filename,
    context.data_folder_path / STRINGIFY(TEST_ID) ".output.expected");
}
