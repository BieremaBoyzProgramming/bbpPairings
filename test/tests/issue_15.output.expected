issue_15: Round #1
issue_15: Round #2
issue_15: Round #3
issue_15: Round #4
issue_15: Round #5
issue_15: Round #6
issue_15: Round #7
issue_15: Round #8
issue_15: Round #9
issue_15: Round #10
issue_15: Round #11

